package.json

```diff
@@ -14,7 +14,7 @@
     "test:ci": "vitest run",
     "lint": "node node_modules/.bin/eslint src",
     "check:ci": "yarn lint && yarn test:ci && yarn tscheck",
-    "pre-commit": "vitest run && yarn run lint",
+    "pre-commit": "yarn run prebuild && vitest run && yarn run lint",
```

Comment:
I think we don't need to run this per each commit. it's enough to just run it per new market addition